### ISSUE TITLE
support validate some keys is grouped

### ISSUE THREAD

**Issue by tjx666** [ota-meshi/eslint-plugin-jsonc#209]
all key starts with build should be grouped.

bad:

```json
"scripts": {
    "preinstall": "",
    "build:design": "",
    "clean": "",
    "build:analyze": "",
    "build": "",
    "preview:design": "",
}
```

good:

```json
"scripts": {
    "preinstall": "",
    "clean": "",
    "build:design": "",
    "build:analyze": "",
    "build": "",
    "preview:design": "",
}
```

**Comment by ota-meshi** [comment]
What options do you suggest?

**Comment by tjx666** [comment]
```javascript
{
    'jsonc/group-keys': [
        error,
        {
            pathPattern: '^scripts$',
            // some package like use script like build-dev not build:dev
            separators: [':', '-']
        }
    ],
}
```

**Comment by ota-meshi** [comment]
I don't understand your suggestion. BTW, you can also create your own plugins to realize your ideas.

**Comment by tjx666** [comment]
I uodate the previous reply. Many node package use the `:` to separate the scripts key, so I think this rule can help us to maintain good style.
